src/networking.c

```diff
@@ -2622,6 +2623,69 @@ NULL
         } else {
             addReplyLongLong(c,-1);
         }
+    } else if (!strcasecmp(c->argv[1]->ptr,"trackinginfo") && c->argc == 2) {
+        addReplyMapLen(c,4);
+
+        /* Flags */
+        addReplyBulkCString(c,"flags");
+        void *arraylen_ptr = addReplyDeferredLen(c);
+        int numflags = 0;
+        addReplyBulkCString(c,c->flags & CLIENT_TRACKING ? "on" : "off");
+        numflags++;
+        if (c->flags & CLIENT_TRACKING_BCAST) {
+            addReplyBulkCString(c,"bcast");
+            numflags++;
+        }
```

Comment:
i would have considered creating a short macro.
one line create the macro, use it 6 time, and undef it. (total of 8 lines).
maybe it'll look better..

not a deal breaker of course...

Reply:
I think local macros to save number of lines generally end up distracting me when reading, but also seem nicer when I'm writing them.